package.json

```diff
@@ -27,6 +27,15 @@
   "author": "Apostrophe Technologies, Inc.",
   "license": "MIT",
   "dependencies": {
+    "@babel/core": "^7.13.16",
```

Comment:
This is a tiptap peerRequisite?

Reply:
hrmm that line is part of https://github.com/apostrophecms/apostrophe/pull/2959/commits/dfa2eb1180b87fce9473c90afe7efc834fad2122

I didn't intentionally add it .. removing it and babel-env builds fine soooo